code/dqe/bycount.q

```diff
@@ -0,0 +1,8 @@
+\d .dqe
+
+bycount:{[tab;bycols]
+  .lg.o[`bycount;"Counting amount of messages received with by clauses applied to column(s) bycols"];
+  $[0>type bycols;
```

Comment:
is `bycols` a symbol or list of symbols? If so be explicit about the expected type:
```suggestion
  $[-11h=type bycols;
```